swupd/hash.go

```diff
@@ -241,3 +173,63 @@ func (h *Hash) Sum() string {
 	hex.Encode(result[:], h.hmac.Sum(nil))
 	return string(result[:])
 }
+
+// GetHashFile calculate the swupd hash for a file in the disk.
+func GetHashFile(filename string) (string, error) {
```

Comment:
Sorry if I'm being too picky...but GetHashFile sounds like it's returning me a - Hash "file", not a string. Would it make more sense to just call it GetHash? or GetFileHash()?

Reply:
Maybe GetHashForFile and GetHashForBytes?

Reply:
@tmarcu You can (almost) never be too picky about exported Names